js/blocks/controls/index.js

```diff
@@ -0,0 +1,17 @@
+import ACBTextControl from './text'
+import ACBTextareaControl from './textarea'
+import ACBCheckboxControl from './checkbox'
+import ACBRadioControl from './radio'
+import ACBRangeControl from './range'
+import ACBSelectControl from './select'
+import ACBToggleControl from './radio'
+
+export default {
+	text: ACBTextControl,
+	textarea: ACBTextareaControl,
+	checkbox: ACBCheckboxControl,
+	radio: ACBRadioControl,
+	range: ACBRangeControl,
+	select: ACBSelectControl,
+	toggle: ACBToggleControl,
+}
```

Comment:
If I were a third-party plugin wanting to extend this list, how would I do that?